examples/ts-client/client_lib/README.md

```diff
@@ -0,0 +1,26 @@
+# Static grpc client generator for Vixen streams
+
+This small package is intended to facilitate the grpc clients construction for consuming Vixen streams. It has dependencies on [@grpc/grpc-js](https://github.com/grpc/grpc-node/tree/master/packages/grpc-js) for the actual network service implementation, and on [protobufjs-cli](https://github.com/protobufjs/protobuf.js/tree/master/cli) for the pre-generated static classes and types for the grpc Messages.
+
+## Updating static generated code
```

Comment:
I just had the thought about Triton stream would be different from someone else's.  In this case each unique stream (with differing proto schema) would need to generate its own sdk.

In the short term, we'll get a dedicated repo for hosting the package but it will be versioned and published against the host version of streams. Then we can loop back here and see if we can create generator of some sort to scaffold sdk. Maybe have template repository people can fork and then has Make and package run commands for doing this part.